DepositRegistrar.sol

```diff
@@ -0,0 +1,306 @@
+// Name registrar using deposits
+
+contract Deed {
+    // The Deed is a contract intended simply to hold ether
+    // It can be controlled only by the registrar and can only send ether back to the owner
+    address public owner;
+    address public bidder;
+    Registrar public registrar;
+    bytes32 public sealedBid;
+    bytes32 public hashedName;
```

Comment:
Yeah, naming things has been hard. This is how I went to it:

Name: the thing we are bidding and own
Hashed Name: the proper name, passed through a sha3 to add anonymity
Deed: the contract that holds information about the property. I believe Deed is a good term

I don't understand what "label" means in this context